hyrax/app/assets/javascripts/hyrax/save_work/required_fields.es6

```diff
@@ -7,80 +11,68 @@ export class RequiredFields {
   }
 
   get areComplete() {
-		this.reload()
```

Comment:
this reload was an add in your code, which isn't needed now that we hook in to the existing events